harmonica/README.md

```diff
@@ -0,0 +1,67 @@
+# Harmonica
+Annotate data with ontology terms.
+
+## Prerequisites
+The prerequisites to run the script are maintained within the `pyproject.toml` file. To update your virtual environment run `poetry install`.
+
+## Usage
+### Python
+```
+python harmonize.py annotate \
+    --config ../config/config.yml \
+    --input_file ../toy_data/raw_data_conditions/conditions_simple.tsv \
+    --output_dir tmp/output/ \
```

Comment:
Is this the right location? Are we running `harmonica` from it's own directory?

Reply:
Yes, currently. I followed the pattern for the `simple_clean`, which is at the same level, but these seem like they should be under `src`. Can you let me know what location you prefer?